bpf/process/bpf_rate.h

```diff
@@ -31,19 +33,30 @@ struct {
 
 static inline __attribute__((always_inline)) bool
 cgroup_rate(struct cgroup_rate_key *key, __u64 ns,
-	    const struct cgroup_rate_settings *settings)
+	    const struct cgroup_rate_settings *settings,
+	    bool *throttle)
```

Comment:
So if I understand correctly, we:
* return `true` if the rate is within our settings
* return `false` if the rate is not within our settings
     * we set throttle to true, if a throttle message is needed?

Reply:
right